src/http-api/routes/bootstrap.js

```diff
@@ -17,14 +16,11 @@ module.exports = (server) => {
   api.route({
     method: '*',
     path: '/api/v0/bootstrap/add',
-    handler: resources.bootstrap.add,
     config: {
-      validate: {
-        query: {
-          arg: Joi.string().required(), // multiaddr to add
-          default: Joi.boolean()
-        }
```

Comment:
Actually, the offending parameter is not documented in the HTTP API: http://docs.ipfs.apiary.io/#reference/bootstrap/add/add does not list `stream-channels`, which is always set to true by js-ipfs-api: https://github.com/ipfs/js-ipfs-api/blob/master/src/request-api.js#L92